Error with the latest version of Dioxus

When using Markdown in the latest version of Dioxus, I get this error :
```
error[E0308]: mismatched types
  --> src/main.rs:8:13
   |
7  | /         rsx! {
8  | |             Markdown {
   | |             ^^^^^^^^ expected fn pointer, found fn item
9  | |                 content: "# Title"
10 | |             }
11 | |         }
   | |_________- arguments to this function are incorrect
   |
   = note: expected fn pointer `fn(&Scoped<'_, _>) -> _`
                 found fn item `for<'a> fn(dioxus_core::scopes::Scope<'a, MarkdownProps<'a>>) -> Option<dioxus_core::nodes::VNode<'a>> {Markdown}`
   = note: when the arguments and return types match, functions can be coerced to function pointers
note: function defined here
  --> /home/riccardo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dioxus-core-0.3.3/src/properties.rs:73:8
   |
73 | pub fn fc_to_builder<'a, A, T: Properties + 'a>(_: fn(Scope<'a, T>) -> A) -> T::Builder {
   |        ^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `reproduce_error` (bin "reproduce_error") due to previous error
```
Here is a code to reproduce this error :
```rust
use dioxus::prelude::*;
use dioxus_markdown::Markdown;

#[allow(non_snake_case)]
fn App(cx: Scope) -> Element {
    render! {
        rsx! {
            Markdown {
                content: "# Title"
            }
        }
    }
}

fn main() {
    dioxus_desktop::launch(App);
}
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,8 +6,8 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-dioxus = { version = "0.2.4", features = ["html"] }
+dioxus = "0.3"
 pulldown-cmark = "0.9.1"
 
 [dev-dependencies]
-dioxus = { version = "0.2.4", features = ["desktop"] }
+dioxus-desktop = "0.3"

--- a/examples/bulma.rs
+++ b/examples/bulma.rs
@@ -2,7 +2,7 @@ use dioxus::prelude::*;
 use dioxus_markdown::Markdown;
 
 fn main() {
-    dioxus::desktop::launch(app);
+    dioxus_desktop::launch(app);
 }
 
 fn app(cx: Scope) -> Element {

